src/app/extensions/quoting/shared/quoting-basket-line-items/quoting-basket-line-items.component.html

```diff
@@ -0,0 +1,18 @@
+<ng-container *ngFor="let item of lineItems$ | async">
+  <div class="row">
+    <div class="col-md-6 my-auto">
+      <h1 *ngIf="item[0] !== 'undefined'">{{ getName(item[0]) | async }}</h1>
+    </div>
```

Comment:
- The ngIf should probably be on the div with the 'row', so it doesn't apply for normal items
- Please introduce a translation key `From Quote {{quoteId}}`
- I don't think the column layout is set up correctly. On smaller devices, the recycle bin icon breaks in a wired way to the next line.